src/userbase-server/setup.js

```diff
@@ -10,19 +10,20 @@ let initialized = false
 const defaultRegion = 'us-west-2'
```

Comment:
Just curious: any specific reason `us-west-2`? Although you conjectured `us-east-1` is the [most reliable AWS region](https://twitter.com/dvassallo/status/1193665216573140992)? Thanks :)

Reply:
Ha! — We use us-west-2 only for development, since most of the people working on this are closer to that region. In production we use us-east-1. 😁